frontend/src/concepts/k8s/NameDescriptionField.tsx

```diff
@@ -42,14 +46,15 @@ const NameDescriptionField: React.FC<NameDescriptionFieldProps> = ({
   return (
     <Stack hasGutter>
       <StackItem>
-        <FormGroup label="Name" isRequired fieldId={nameFieldId}>
+        <FormGroup label="Name" isRequired fieldId={nameFieldId} helperText={nameHelperText}>
```

Comment:
You may also want to add the invalid state to the helper text following the PF examples http://v4-archive.patternfly.org/v4/components/form#invalid
```suggestion
        <FormGroup
          label="Name"
          isRequired
          fieldId={nameFieldId}
          helperText={nameHelperText}
          helperTextInvalid={nameHelperText}
          helperTextInvalidIcon={<ExclamationCircleIcon />}
          validated={!isNameValid ? 'error' : undefined}
        >
```

Reply:
cc @uidoyen

Reply:
This change can also be reverted based on the new updates you give.